File: apps/institution-api/Models/Dto/SpecializationYearlyCostDto.cs
using System;

namespace ProviderApi.Models.Dto
{
  public class SpecializationYearlyCostDto
  {
    public int SpecializationId { get; set; }
    public int SessionOfStudyId { get; set; }
    public DateTime? SessionStartDate { get; set; }
    public DateTime? SessionEndDate { get; set; }
    public decimal? Cost { get; set; }
  }
}

File: libs/common/models/specialization-yearly-cost-dto.ts



export class SpecializationYearlyCostDto {
    public specializationId: number = 0;
    public sessionOfStudyId: number = 0;
    public sessionStartDate: Date;
    public sessionEndDate: Date;
    public cost: number;

    constructor(init? : Partial<SpecializationYearlyCostDto>) {
        
        Object.assign(this, init);
    }
}

